lib/internal/net.js

```diff
@@ -9,3 +9,9 @@ function isLegalPort(port) {
     return false;
   return +port === (port >>> 0) && port >= 0 && port <= 0xFFFF;
 }
+
+
+function assertPort(port) {
+  if (typeof port !== 'undefined' && !isLegalPort(port))
```

Comment:
I don't think so. The changes in #5733 make `isLegalPort()` validate the value of `port`. `undefined` isn't a valid port, and is specific to this use case.